src/main/java/com/powsybl/openloadflow/util/ComplexMatrix.java

```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright (c) 2023, Jean-Baptiste Heyberger <jbheyberger at gmail.com>
+ * Copyright (c) 2023, Geoffroy Jamgotchian <geoffroy.jamgotchian at gmail.com>
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.openloadflow.util;
+
+import com.powsybl.math.matrix.DenseMatrix;
+import com.powsybl.math.matrix.MatrixException;
+import net.jafama.FastMath;
+import org.apache.commons.math3.complex.Complex;
+
+/**
+ * @author Jean-Baptiste Heyberger <jbheyberger at gmail.com>
+ */
+public class ComplexMatrix {
```

Comment:
Can we imagine having this class in `powsybl-core`? There is a lot of small code parts using complex all over Powsybl. @zamarrenolm could be interested here.

Reply:
PR is merged, rework needed later.